index.html

```diff
@@ -48,13 +48,13 @@ <h1 class="title">COVID-19 Visualizer</h1>
       </div>
       <div class="marker-controls">
         <label class="marker-checkbox" for="markers">
```

Comment:
We probably should add a tooltip (`"title"` attribute) to each label element: "Show one dot per confirmed case or reported death", "Show a heatmap of confirmed cases per capita by county", "Show a heatmap of reported deaths per capita by county".